winterfell/webapp/src/js/claimSelectFormsCtrl.js

```diff
@@ -1,13 +1,23 @@
 var app = angular.module('vetafiApp');
-app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', 'formTemplateService', '$stateParams',
-  function($scope, claimService, formTemplateService, $stateParams) {
+app.controller('claimSelectFormsCtrl', ['$scope', 'claimService', '$stateParams',
+  function($scope, claimService, $stateParams) {
     $scope.claimId = $stateParams.claimId;
-    $scope.forms = formTemplateService;
+    $scope.allForms = [
```

Comment:
oh I see, `formTemplateService` is in `formly-fields.js`. I got errors because it wasn't being included in the angular app correctly. Maybe because it is on the same level as `index.jade` / `index.html`? Was that on purpose?